common/lib/dependabot/shared_helpers.rb

```diff
@@ -130,19 +131,20 @@ def self.run_helper_subprocess(command:, function:, args:, env: nil,
       return response["result"] if process.success?
 
       raise HelperSubprocessFailed.new(
-        message: response["error"],
+        message: "#{response['error']} failed with command (#{error_context[:command]})",
```

Comment:
Could there be any PII logged in the command that we're running that we should avoid logging?

Reply:
@jurre , All these logs will be going through the new endpoint and in the new endpoint, we are configuring failbot to not send the capture logs to sentry. The logs will be going to splunk and GitHub data warehouse. 
https://github.com/github/dependabot-api/pull/4213/files#diff-8988e11e084c4f99d07033e93421b1119a84ee3d9f9e3c9a94be43959bc91420R74-R77